### ISSUE TITLE
Update Angular.js to Angular

### ISSUE THREAD

**Issue by ShinDarth** [vert-x3/vertx-guide-for-java-devs#56]
Chapter 10 "Client-side web application using AngularJS." still refers to the old [AngularJS](https://angularjs.org/) (which is no longer actively developed). What about updating it to the new [Angular](https://angular.io/)? Would PRs be accepted to do it?

**Comment by jponge** [comment]
Yes PRs are welcome!

**Comment by FrancescoBorzi** [comment]
@jponge yes I was looking at that piece of code and I'm wondering where we are referencing the `webroot` directory? sorry I'm completely new to Vert.x

**Comment by jponge** [comment]
This is by default, but this can be overridden https://vertx.io/docs/apidocs/io/vertx/ext/web/handler/StaticHandler.html#create-java.lang.String-

**Comment by FrancescoBorzi** [comment]
ok, thanks a lot for your answer!

**Comment by FrancescoBorzi** [comment]
@jponge what's the best way to enable CORS in step-9 ? I need it for development purposes

**Comment by FrancescoBorzi** [comment]
@jponge  I've finished implementing the demo app for `step-9`.

You can find the source code here: https://github.com/FrancescoBorzi/vertx-guide-client-app

I've tried to keep the behaviour, code and logic similar to the old app whenever possible.

Before I start editing the book chapter, I kindly ask you to **review the code**. If you think it's good enough, I'll start adjusting the book chapter.

**Comment by jponge** [comment]
I am no modern Angular expert, so it looks good to me.

That's a lot of files, can't it be made smaller?

**Comment by FrancescoBorzi** [comment]
@jponge we could get rid of `e2e` directory and maybe also of the unit tests (`*.spec.ts`) since we don't really need it. Then I don't know what else we can remove. What do you say?
